core/gethwrappers/keystone/go_generate.go

```diff
@@ -0,0 +1,7 @@
+// Package gethwrappers provides tools for wrapping solidity contracts with
+// golang packages, using abigen.
+package gethwrappers
+
+// Keystone
+
+//go:generate go run ../generation/generate/wrap.go ../../../contracts/solc/v0.8.19/KeystoneForwarder/KeystoneForwarder.abi ../../../contracts/solc/v0.8.19/KeystoneForwarder/KeystoneForwarder.bin KeystoneForwarder forwarder
```

Comment:
NOTE: getting the solc compiler wrapper working was really tedious so I ended up using `forge build --extra-output-files abi` then `abigen --abi ../../../contracts/foundry-artifacts/KeystoneForwarder.sol/KeystoneForwarder.abi.json --pkg forwarder --out generated/forwarder/forwarder.go`